scripts/pi-hole/php/footer.php

```diff
@@ -72,20 +57,20 @@
         </div>
 
         <div class="row row-centered text-center version-info">
-            <div class="col-xs-12 col-sm-8 col-md-6">
+            <div class="col-xs-12 col-sm-12 col-md-10">
                 <?php if (isset($core_commit) || isset($web_commit) || isset($FTL_commit)) { ?>
                 <ul class="list-unstyled">
-                    <?php if ($dockerTag) { ?> <li><strong>Docker Tag</strong> <?php echo $dockerTag; ?></li> <?php } ?>
+                    <?php if ($docker_current) { ?> <li><strong>Docker Tag</strong> <?php echo $docker_current; ?></li> <?php } ?>
                     <li><strong>Pi-hole</strong> <?php echo $coreVersionStr; ?></li>
                     <li><strong>FTL</strong> <?php echo $ftlVersionStr; ?></li>
                     <li><strong>Web Interface</strong> <?php echo $webVersionStr; ?></li>
                 </ul>
                 <?php } else { ?>
                 <ul class="list-inline">
-                    <?php if ($dockerTag) { ?>
+                    <?php if ($docker_current) { ?>
                     <li>
                         <strong>Docker Tag</strong>
-                        <a href="<?php echo $dockerReleasesUrl.'/'.$dockerTag; ?>" rel="noopener" target="_blank"><?php echo $dockerTag; ?></a>
+                        <a href="<?php echo $dockerReleasesUrl.'/'.$docker_current; ?>" rel="noopener" target="_blank"><?php echo $docker_current; ?></a>
```

Comment:
```suggestion
                        <a href="<?php echo $dockerReleasesUrl.'/'.$docker_current; ?>" rel="noopener" target="_blank"><?php echo $docker_current; ?></a>
                        <?php if ($docker_update) { ?> &middot; <a class="lookatme" lookatme-text="Update available!" href="<?php echo $dockerReleasesUrl.'/latest'; ?>" rel="noopener" target="_blank">Update available!</a><?php } ?>
```

Reply:
And actually "while you're at it".. you may as well add in the suppression that I was planning to add in my PR

Reply:
I think I did it in the `update_checker.php` code.

Reply:
Ah, right ok - yep I see the logic there. In which case I think my PR is null and void?

Reply:
But I think you should still add the `lookatme` thingy 
![image](https://user-images.githubusercontent.com/1998970/186508761-0d6f6683-f7ed-45b5-8af6-c80e41165e65.png)

Reply:
I will make the changes here.

After that, your PR will be unnecessary.